core-libs/setup/ssr/logger/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './loggers';
+export * from './services';
```

Comment:
[MINOR]

TBH I'm not sure what's the difference, but we usually export from `/xxx/index`. I recall in the past it caused problems when building libraries with ng-packgr. Not sure if it's still relevant problem today
```suggestion
export * from './loggers/index';
export * from './services/index';
```

Reply:
done in https://github.com/SAP/spartacus/pull/17538